packs/curtain-call-bestiary/book-2-singer-stalker-skinsaw-man/vorens.json

```diff
@@ -246,14 +246,22 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>Vorens is adept at exploiting an opponent's weaknesses in a duel. If he wins initiative in a duel round, he deals an extra 2d6 precision damage with Strikes made before the beginning of his next turn.</p>"
+                    "value": "<p>Vorens is adept at exploiting an opponent's weaknesses in a duel. If he wins initiative in a duel round, he deals an extra 2d6 precision damage with Strikes made before the beginning of his next turn.</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Adroit Duelist]</p>"
```

Comment:
This is maybe doable automatically by having a toggle for being in a duel and then just if his initiative is higher than the other person's he gets the damage. Since in a duel initiative rerolls at the end of each round.

What do you think?